### DISCUSSION TITLE
"There was an error pushing commits to GitHub" for LFS-enabled repo

### DISCUSSION THREAD

**Discussion by lupine-dev** [community/community#21646] [New to GitHub]
<p>When importing a public LFS-enabled GitHub repository using <a href="http://GitHub.com" rel="noopener nofollow ugc">GitHub.com</a>’s built-in import tool, after choosing to include the LFS files, I get the following message:</p>
<blockquote>
<p>We found an error during import</p>
<p>We recommend restarting the import process, but if you are still having trouble contact support.</p>
<p>There was an error pushing commits to GitHub.</p>
<p>(Cancel) (Restart)</p>
</blockquote>
<p>Upon clicking “Restart”, the same error occurs.</p>
<p>The public repository I am trying to import is the following:</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/Apress/repo-with-large-file-storage" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/4030205/181065988-6a65156c-481f-4764-b319-eaffbf3951c4.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/1/a184dd511557f9e156dae4b298467659a835bca9_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/a/1/a184dd511557f9e156dae4b298467659a835bca9_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/a/1/a184dd511557f9e156dae4b298467659a835bca9.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/1/a184dd511557f9e156dae4b298467659a835bca9_2_10x10.png"></div>

<h3><a href="https://github.com/Apress/repo-with-large-file-storage" target="_blank" rel="noopener nofollow ugc">Apress/repo-with-large-file-storage</a></h3>

  <p><span class="label1">An example repo using Git Large File Storage (LFS)</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>I am trying to import it into a private repository for testing purposes (to test out git-lfs with authentication).</p>
<p>Is this error a known issue?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>Yes - this is a known issue, and I’ve just added this to the issue so we can document it better.</p>
<p>Our import tool is more for importing from other version control systems than it is for git repositories. As such, it doesn’t know what to do with lfs in the origin repository.</p>
<p>It can recognise there are large files to handle with lfs, but when those large files are already handled as lfs in the origin repository, it all falls down.</p>
<p>It’s probably far simpler to clone the repository to your local system and then push it as a new repository in your GitHub account.</p>
<p>This article here might help:</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/duplicating-a-repository#mirroring-a-repository-that-contains-git-large-file-storage-objects" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/duplicating-a-repository#mirroring-a-repository-that-contains-git-large-file-storage-objects" target="_blank" rel="noopener">Duplicating a repository - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/duplicating-a-repository#mirroring-a-repository-that-contains-git-large-file-storage-objects" target="_blank" rel="noopener">//docs.github.com/en/github/creating-cloning-and-archiving-repositories/duplicating-a-repository</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
